docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -1,135 +1,135 @@
 ---
-title: How to Open a Pull Request
+title: Como Abrir uma Pull Request
 ---
 
-A big part of contributing to open source is submitting changes to a project: improvements to source code or tests, updates to docs content, even typos or broken links. This doc will cover what you need to know to **open a pull request** in Gatsby.
+Uma boa parte de contribuição para open source consiste em enviar alterações para um projeto: melhorias no código-fonte ou nos testes, atualizações no conteúdo da documentação, ou até mesmo erros de digitação ou links quebrados. Este documento abordará o que você precisa saber para **abrir uma pull request** no Gatsby.
```

Comment:
colocar em itálico open source => _open source_ 
Alterar artigo "uma" => "um"

```suggestion
Uma boa parte de contribuição para _open source_ consiste em enviar alterações para um projeto: melhorias no código-fonte ou nos testes, atualizações no conteúdo da documentação, ou até mesmo erros de digitação ou links quebrados. Este documento abordará o que você precisa saber para **abrir um pull request** no Gatsby.
```

Reply:
https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/f5af213ee6289a6e0645e9f7d7ef16f0b3cfe22d
https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/3b3c21095fb45f6be256aa235cc1b8677bc185fa